src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
 server.servlet.context-path=/team3
+
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=fae.nachrichtensystem@gmail.com
+spring.mail.password=qpkesrtalaxkpkiv
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.starttls.enable=true
```

Comment:
Wäre das ein Fall für die docker-compose?

Reply:
Hast du dabei an etwas bestimmtes gedacht?
Ich habe mich an [diesem Tutorial](https://www.baeldung.com/spring-email) orientiert und dort ist es üblich die Mail Konfigurationen in die Spring Konfigurationsdatei zu packen. Lasse mich da aber auch gerne belehren falls es andere Best Practices gibt :)

Reply:
Eventuell liege ich da falsch aber ich glaube ich habe mal gelesen, dass man Sachen wie Passwörter und so als Environment Variable zur Laufzeit reingibt, weil man die nicht in seinem Docker image haben will.

In unserem Kontext ist das wahrscheinlich nicht so wichtig und das ist vermutlich ja auch nicht dein privater Mailaccount :-D

Reply:
Macht definitiv Sinn, werde mich da mal ein wenig einlesen.
Und nein, die Mail Adresse habe ich extra dafür erstellt und das Passwort ist auch "nur" ein Applikations-Passwort

Reply:
[Docker secrets](https://docs.docker.com/engine/swarm/secrets/) könnte man sich mal anschauen

Reply:
Oh okay, schade!